ui/lib/pki/addon/components/page/pki-configuration-details.hbs

```diff
@@ -30,4 +61,37 @@
 <InfoTableRow @label="Default lease TTL" @value={{@mountConfig.config.defaultLeaseTtl}} />
 <InfoTableRow @label="Max lease TTL" @value={{@mountConfig.config.maxLeaseTtl}} />
 <InfoTableRow @label="Allowed managed keys" @value={{or @mountConfig.config.allowedManagedKeys "None"}} />
-<div class="has-top-margin-l"></div>
\ No newline at end of file
+<div class="has-top-margin-l"></div>
+
+{{#if this.showDeleteAllIssuers}}
+  <Modal
```

Comment:
I think there's actually s specific `<ConfirmationModal>` component for this use case. (Which might solve the design discrepancy below)

Reply:
I'll ask Yongqi if we want the modal to look like that. In the pki designs, it doesn't look like a `ConfirmationModal`

Reply:
Update: spoke to Yongqi and she agrees that using ConfirmationModal makes sense here. I also updated the button text to be Confirm instead of delete since the user is typing `delete-all` it doesn't sense for the button to say delete when it really is just to confirm the deletion. It is also the same behavior in the replication demotion.

<img width="739" alt="Screenshot 2023-03-31 at 10 34 08 AM" src="https://user-images.githubusercontent.com/30884335/229197478-45206ec1-b1a1-41d1-94b9-d65b9cb5e4b0.png">